### PR TITLE
feat: ability to embed content from site

### PR DESCRIPTION
**Pull request by devodii**

### CODE REVIEW

File: components/Embed.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface Props {
+  src: string;
+  title: string;
```
Comment (BleedingDev) [inline components/Embed.tsx:5]: **praise:** Anything besides src should be optional and have default params. Title empty, width 100 %, height I think 300px.

File: puck.config.tsx [hunk 2]
```diff
@@ -261,6 +263,22 @@ export const config = {
...
+      fields: {
+        height: { type: "number" },
+        src: { type: "text" },
+        title: { type: "text" },
+        width: { type: "number" },
+      },
+
+      defaultProps: {
+        height: 100,
+        src: "https://google.com",
```
Comment (BleedingDev) [inline puck.config.tsx:276]: **praise:** Does Google actually load in an iframe? I think they block it.

**Review by BleedingDev — CHANGES_REQUESTED** [review]

File: puck.config.tsx (hunk 2 above)
Comment (devodii) [reply puck.config.tsx:278]: Yeah, they did.
Will correct that.

File: components/Embed.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export function Embed({ src, title, width, height }: Props) {
+  return (
+    <iframe
+      referrerPolicy="no-referrer"
+      sandbox="allow-forms allow-scripts"
+      src={src}
+      title={title}
+      width={width ?? "100%"}
```
Comment (BleedingDev) [inline components/Embed.tsx:17]: **issue:** user shouldn't have to write percentage. We need the user to write just the number and then add %.

```typescript
width={`${width ?? 100}%`}
```

File: components/Embed.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      height={height ?? "300px"}
```
Comment (BleedingDev) [inline components/Embed.tsx:18]: **issue:** same problem here, we don't want to force user writing more than necessary, which `px` is.

**Review by BleedingDev — CHANGES_REQUESTED** [review]

File: components/Embed.tsx (hunk 3 above)
Comment (devodii) [reply components/Embed.tsx:17]: praise: Makes sense!

File: components/Embed.tsx (hunk 4 above)
Comment (devodii) [reply components/Embed.tsx:18]: So, we leave it hardcoded?
No input from user?

Comment (BleedingDev) [reply components/Embed.tsx:18]: No. The problem is, that we ask user for `number`, but then we add it as a number, not as `${number}px`.

File: components/Embed.tsx (hunk 3 above)
Comment (BleedingDev) [reply components/Embed.tsx:17]: Great, can we fix it? :)

**Review by BleedingDev — CHANGES_REQUESTED** [review]

**Review by BleedingDev — APPROVED** [review]

**Review by BleedingDev — APPROVED** [review]

**Pull request merged**